src/compiler/triage-meeting.md

```diff
@@ -2,11 +2,6 @@
 
 ## What is it?
```

Comment:
Maybe remove this header too?

Reply:
I intended to delete this file and its content is included in the more general meetings page.